src/box/lua/feedback_daemon.lua

```diff
@@ -212,7 +212,7 @@ local cached_schema_version = 0
 local cached_schema_features = {}
 
 local function fill_in_schema_stats(features)
-    local schema_version = box.internal.schema_version()
+    local schema_version = box.schema_version()
```

Comment:
Come to think of it, we usually return information like this to Lua via `box.info`.

Reply:
Exported schema version to Lua via `box.info.schema_version`.